package.json

```diff
@@ -55,7 +55,6 @@
         "@coveo/semantic-monorepo-tools": "1.7.2",
         "@sindresorhus/slugify": "2.2.0",
         "aws-sdk": "2.1379.0",
-        "axios": "0.27.2",
```

Comment:
I don't see any change to pnpm-lock.yaml, are you missing an install?

Reply:
Thought I did, I'll do it again